src/main/java/com/google/devtools/build/lib/bazel/bzlmod/GsonTypeAdapterUtil.java

```diff
@@ -283,6 +301,68 @@ public Optional<T> read(JsonReader jsonReader) throws IOException {
     }
   }
 
+  /**
+   * Converts Guava tables into a JSON array of 3-tuples (one per cell). Each 3-tuple is a JSON
```

Comment:
I don't understand this comment: can you provide a small example JSON to clarify?

Reply:
added a small snippet.